src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/CheckSafe.tsx

```diff
@@ -51,6 +54,18 @@ const MSG = defineMessages({
     id: 'dashboard.AddExistingSafeDialog.CheckSafe.safe',
     defaultMessage: 'Safe',
   },
+  callout: {
+    id: 'dashboard.AddExistingSafeDialog.CheckSafe.callout',
+    defaultMessage: '<span>Important!</span>  Read the instuctions first.',
```

Comment:
@ArmandoGraterol We have the typo here still "instuctions: should be "instructions"